app/bundles/CampaignBundle/Views/Event/preview.html.twig

```diff
@@ -0,0 +1,30 @@
+{% if route is not defined or route is empty %}
+  {% set route = 'mautic_campaignevent_action' %}
+{% endif %}
+
+{% set eventType = event['eventType'] %}
+{% set eventLogic = '' %}
+
+{# Show ID in dev mode to help with debugging #}
+{% set eventName = ('dev' == constant('MAUTIC_ENV')) ? event['name']~' <small>'~event.id~'</small>' : event['name'] %}
+
+{% if update is not defined or update is empty %}
+<div id="CampaignEvent_{{ event['id'] }}" data-type="{{ event['eventType'] }}" class="draggable list-campaign-event list-campaign-{{ event['eventType'] }}" data-event="{{ event['type'] }}" data-event-id="{{ event['id'] }}">
+{% endif %}
+
+    <div class="campaign-event-content">
+        <div><span class="campaign-event-name ellipsis">{{ eventName|raw }}</span></div>
```

Comment:
Please do not use the `raw` filter. Use `purify` instead to avoid XSS vulnerability